lms/templates/public_video.html

```diff
@@ -0,0 +1,18 @@
+<%page expression_filter="h"/>
+<%inherit file="courseware/courseware-chromeless.html"/>
+
+<%block name="head_extra">
+<!-- OpenGraph tags  -->
+<meta data-rh="true" property="og:type" content="website">
+<meta data-rh="true" property="og:site_name" content="edX">
+<meta data-rh="true" property="og:title" content=${video_title}>
+<meta data-rh="true" property="og:description" content="${video_description}">
+<meta data-rh="true" property="og:image" content="${video_thumbnail}">
```

Comment:
Can you add `og:video:url` and `og:video:secure_url`? Most of the known website use that.

Reply:
I'll toss that into another ticket. DO we know for sure that gets ingested by other platforms? For now we don't need it for twitter